packages/runtime/src/types/transport/RelationshipDTO.ts

```diff
@@ -32,12 +33,14 @@ export interface RelationshipAuditLogEntryDTO {
 
 export interface RelationshipAuditLogDTO extends Array<RelationshipAuditLogEntryDTO> {}
 
+export type RelationshipCreationContentDTO = RelationshipCreationContentJSON | ArbitraryRelationshipCreationContentJSON;
```

Comment:
@jkoenig134 didn't like the previous version of RelationshipCreationContentContainingResponse, and I'm not seeing a way of adjusting RelationshipCreationContentDTO without it looking ugly and misleading in the direction of arbitraryContent right now. I would like to keep the type safety rather than revert to any